features/step_definitions/government_frontend_steps.rb

```diff
@@ -0,0 +1,36 @@
+Then /^I should see a radio button for "(.*?)"$/ do |value|
+  # We set visible to false since our radios are styled labels
+  page.body.should have_field("option", with: value, visible: false)
+end
+
+Then /^I should see a continue button$/ do
+  # We set visible to false since our radios are styled labels
+  expect(page).to have_selector('button', text: "Continue")
+end
+
+When /^I choose "(.*?)"$/ do |option|
+  choose(option, visible: false)
+  click_button "Continue"
+end
+
+Then /^I should be redirected to "(.*?)"$/ do |url_or_path|
+  if url_or_path.start_with?('http')
+    # Some urls do a double redirect, we only care about the URL we end up on.
+    wait_until { page.current_url == url_or_path }
+    expect(page.current_url).to(eq(url_or_path))
+  else
+    page.current_path.should eq(url_or_path)
```

Comment:
For consistency, should this be `expect(page.current_url).to(eq(url_or_path))` as above?